MoltenVK/MoltenVK/Utility/MVKFoundation.h

```diff
@@ -384,24 +387,24 @@ static constexpr bool mvkVkComponentMappingsMatch(VkComponentMapping cm1, VkComp
 
 /** Rounds the value to nearest integer using half-to-even rounding. */
 static inline double mvkRoundHalfToEven(const double val) {
-	return val - std::remainder(val, 1.0);	// remainder() uses half-to-even rounding, and unfortunately isn't constexpr until C++23.
+	return val - std::remainder(val, 1.0);	// remainder() uses half-to-even rounding, but unfortunately isn't constexpr until C++23.
 }
 
 /** Returns whether the value will fit inside the numeric type. */
 template<typename T, typename Tval>
-const bool mvkFits(const Tval& val) {
+static constexpr const bool mvkFits(const Tval& val) {
```

Comment:
The `const` on the return type is useless.

Reply:
Thanks. Copy-pasto during mods.